crates/core/tedge_api/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+#![allow(dead_code)]
```

Comment:
Do we still need this? I think we can remove this already, because everything we have is either public or used... isn't it?